packages/preview/unequivocal-ams/0.1.0/template.typ

```diff
@@ -0,0 +1,250 @@
+  set enum(indent: 24pt, body-indent: 5pt)
+  show link: set text(font: "New Computer Modern Mono")
+
+  // Configure equations.
+  show math.equation: set block(below: 8pt, above: 9pt)
+  show math.equation: set text(weight: 400)
+
+  // Configure citation and bibliography styles.
+  set bibliography(style: "springer-mathphys", title: [References])
+
+  show figure: it => {
+    show: pad.with(x: 23pt)
+    set align(center)
+
+    v(12.5pt, weak: true)
+
+    // Display the figure's body.
+    it.body
+
+    // Display the figure's caption.
+    if it.has("caption") {
+      // Gap defaults to 17pt.
+      v(if it.has("gap") { it.gap } else { 17pt }, weak: true)
+      smallcaps(it.supplement)
+      if it.numbering != none {
+        [ ]
+        it.counter.display(it.numbering)
+      }
+      [. ]
+      it.caption.body
+    }
+
+    v(15pt, weak: true)
+  }
+
+  // Theorems.
+  show figure.where(kind: "theorem"): it => block(above: 11.5pt, below: 11.5pt, {
+    strong({
+      it.supplement
+      if it.numbering != none {
+        [ ]
+        counter(heading).display()
+        it.counter.display(it.numbering)
+      }
+      [.]
+    })
+    emph(it.body)
+  })
+
+  // Display the title and authors.
+  v(35pt, weak: true)
+  align(center, upper({
+    text(size: large-size, weight: 700, title)
+    v(25pt, weak: true)
+    text(size: footnote-size, author-string)
+  }))
+
+  // Configure paragraph properties.
+  set par(first-line-indent: 1.2em, justify: true, leading: 0.58em)
+  show par: set block(spacing: 0.58em)
+
+  // Display the abstract
+  if abstract != none {
+    v(20pt, weak: true)
+    set text(script-size)
+    show: pad.with(x: 35pt)
+    smallcaps[Abstract. ]
+    abstract
+  }
+
+  // Display the article's contents.
+  v(29pt, weak: true)
+  body
+
+  // Display the bibliography, if any is given.
+  if bibliography-file != none {
+    show bibliography: set text(8.5pt)
+    show bibliography: pad.with(x: 0.5pt)
+    bibliography-file
+  }
+
+  // The thing ends with details about the authors.
+  show: pad.with(x: 11.5pt)
+  set par(first-line-indent: 0pt)
+  set text(7.97224pt)
+
+  for author in authors {
+    let keys = ("department", "organization", "location")
+
+    let dept-str = keys
+      .filter(key => key in author)
+      .map(key => author.at(key))
+      .join(", ")
+
+    smallcaps(dept-str)
+    linebreak()
+
+    if "email" in author [
+      _Email address:_ #link("mailto:" + author.email) \
+    ]
+
+    if "url" in author [
+      _URL:_ #link(author.url)
+    ]
+
+    v(12pt, weak: true)
+  }
+}
+
+// The ASM template also provides a theorem function.
+#let theorem(body, numbered: true) = figure(
+  body,
+  kind: "theorem",
+  supplement: [Theorem],
+  numbering: if numbered { "1" },
+)
+
+// And a function for a proof.
+#let proof(body) = block(spacing: 11.5pt, {
+  emph[Proof.]
+  [ ] + body
+  h(1fr)
+  box(scale(160%, origin: bottom + right, sym.square.stroked))
+})
```

Comment:
Files should always end with LF character if it doesn't break some workflow (rare cases). (Some text editors can automatically add the last LF, but others can't.)